packages/block-library/src/column/block.json

```diff
@@ -11,6 +11,9 @@
 		},
 		"width": {
 			"type": "string"
+		},
+		"templateLock": {
+			"type": "string"
```

Comment:
Shouldn't this be a boolean?

Reply:
Hi @ZebulanStanphill, the template lock attribute can contain values like "all", and "insert" which are strings.